homeassistant/components/comelit/coordinator.py

```diff
@@ -38,43 +45,82 @@ def __init__(self, hass: HomeAssistant, host: str, port: int, pin: int) -> None:
         device_registry.async_get_or_create(
             config_entry_id=self.config_entry.entry_id,
             identifiers={(DOMAIN, self.config_entry.entry_id)},
-            model=BRIDGE,
-            name=f"{BRIDGE} ({self.api.host})",
-            **self.basic_device_info,
+            model=device,
+            name=f"{device} ({self._host})",
+            manufacturer="Comelit",
+            hw_version=self._hw_version,
         )
 
-    @property
-    def basic_device_info(self) -> dict:
-        """Set basic device info."""
-
-        return {
-            "manufacturer": "Comelit",
-            "hw_version": "20003101",
-        }
-
-    def platform_device_info(self, device: ComelitSerialBridgeObject) -> dr.DeviceInfo:
+    def platform_device_info(
+        self,
+        object_class: ComelitVedoAreaObject
+        | ComelitVedoZoneObject
+        | ComelitSerialBridgeObject,
```

Comment:
Are all these three used right now?

Reply:
2 out of 3, "Zone" will be used by sensor.

Reply:
Can we implement this in that PR?